.github/CODEOWNERS

```diff
@@ -515,7 +515,7 @@ playwright.config.ts @grafana/plugins-platform-frontend
 /public/app/features/folders/ @grafana/grafana-search-navigate-organise
 /public/app/features/inspector/ @grafana/dashboards-squad
 /public/app/features/invites/ @grafana/sharing-squad
-/public/app/features/library-panels/ @grafana/dashboards-squad
```

Comment:
we have a section in this file with all sharing-related. Feel free to move it if you want
https://github.com/grafana/grafana/blob/main/.github/CODEOWNERS#L683